dist/css/style.css

```diff
@@ -867,6 +868,13 @@ h1 {
   height: 1.4rem;
 }
 
+  .btn--green {
+    background-color: #fff;
+    border: 1px solid #396df2;
+    border-radius: 8px;
+    margin-left: 20px;
+  }
+
 /* Skills */
 .about-self {
   width:  5rem;
```

Comment:
Please kindly implement the mobile view of the popup as it is shown in the Figma template, you are to ensure your design is responsive and looks as close as possible to the Figma template given for the mobile screen 😄 

Figma template :
![image](https://github.com/mohashyne/MSalyhu_portfolio/assets/20537712/44047dfa-d717-4230-93b5-6401b778ba32)

Your design :
![image](https://github.com/mohashyne/MSalyhu_portfolio/assets/20537712/53be9d81-63c6-41fd-b596-fea9f4cf5f18)

Reply:
# changes implemented

The mobile view pop-up is now showing correctly as you requested. 👍